server/src/main/java/org/opensearch/cluster/service/ClusterManagerThrottlingKeys.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster.service;
+
+/**
+ * Class for maintaining all cluster manager throttling key at one place.
+ */
+public final class ClusterManagerThrottlingKeys {
```

Comment:
We don't really explicit keys since all we need to do is guarantee uniqueness. What would be better if we just have registration take care of assigning unique keys. For instance its possible for one action to still use an already in-use key. Having the Throttler handler registration would certainly be a better choice

Reply:
Throttler will not be aware of the which unique key to assign, and we want to keep tasks name and keys consistent so we can get the aggregated task count from the map which we are maintaining.

So that’s why caller will pass the key as well in registration, while throttler will validate that those keys are unique.